src/components/field-label/field-label.js

```diff
@@ -4,9 +4,9 @@ import { Icon } from '@fluentui/react/lib/Icon';
 
 import { fieldLabelContainer, labelStyle, requiredField, toolTipContainer } from './field-label.style';
 
-export const FieldLabel = ({ children, className, required, tooltip }) => (
-  <div className={fieldLabelContainer}>
-    <div className={cx(labelStyle, { [requiredField]: required }, className)}>{children}</div>
+export const FieldLabel = ({ children, className, required, tooltip, ...rest }) => (
```

Comment:
I'm not sure whether we can have a more meaningful name here.

Reply:
I agree. I will probably do some reorganization in the future, where I might also tidy up components like this